charts/verdaccio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A lightweight private npm proxy registry (sinopia fork)
 name: verdaccio
-version: 1.1.0
+version: 1.1.1
```

Comment:
Not sure whether this is just a patch, according what I read about this feature. I am waiting more opinions.

Reply:
I was a little bit confused here too. I'm fine with minor version bump

Reply:
I thought this might have been added by mistake because condition is already in `_helpers.tpl`, so I made patch bump